TombEngine/Objects/TR5/tr5_objects.cpp

```diff
@@ -455,15 +456,15 @@ static void StartEntity(ObjectInfo *obj)
 	obj = &Objects[ID_GUARD_LASER];
 	if (obj->loaded)
 	{
-		obj->initialise = InitialiseGuardLaser;
+		obj->initialise = InitialiseHeavyGuard;
 		obj->collision = CreatureCollision;
-		//obj->control = GuardControlLaser;
+		obj->control = HeavyGuardControl;
 		obj->shadowType = ShadowMode::All;
+		obj->HitRoutine = HeavyGuardHit;
 		obj->biteOffset = 0;
 		obj->HitPoints = 24;
 		obj->pivotLength = 50;
 		obj->radius = 128;
-		obj->explodableMeshbits = 4;
```

Comment:
I saw the discussion of this being removed in Discord, but does it mean that twogun can't be killed at all, or just prevent his head from exploding during headshot?

Reply:
it prevent his head from exploding during headshot. twogun will be killable with the new hitroutine function. Hitroutine does not work with lasersight yet.

Reply:
added hitroutine with lasersight